content/release-10-17-0/applications-10-17-0.md

```diff
@@ -108,26 +106,45 @@ Fix Version</th>
 <tr>
 <td>Java SDK</td>
 <td>Fixed an issue with the <code>AlarmRepresentation</code> comparison in the <code>equals</code> and <code>hashcode</code> methods. Now, instead of comparing the full <code>ManagedObjectRepresentation</code> as the source of alarms, Cumulocity IoT only compares the managed objects ID. There is no change in <code>ManagedObjectRepresentation</code>.</td>
-
 <td>MTM-49181</td>
 <td>10.16.31.0</td>
 </tr>
 
+<tr>
+<td>Microservice SDK</td>
+<td>The Microservice SDK is now based on Spring Boot 2.7.6, using Spring Security 5.7.5.</td>
+<td>MTM-49761</td>
+<td>10.16.248.0</td>
+</tr>
+
+<tr>
+<td>Microservice SDK</td>
+<td>The Microservice SDK is now based on Spring Boot 2.7.6, using Spring Framework 5.3.24.</td>
+<td>MTM-48361</td>
+<td>10.16.232.0</td>
+</tr>

```

Comment:
could those two be condensed into one?

Reply:
No, we need to keep them separately as they refer to different tickets.